### PR TITLE
Adding HINTS.md file to Zebra and Sum-of-Multiple Exercises

### PR DESCRIPTION
**Pull request by GalaDe**

### CODE REVIEW

File: exercises/zebra-puzzle/HINTS.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### Hints
```
Comment (ErikSchierboom) [inline exercises/zebra-puzzle/HINTS.md]: There should be two hashes.

**Comment by ErikSchierboom** [comment]
Would you mind creating a separate PR for each changes file next time? 

Additionally, the commit history looks a bit odd. Something probably went wrong with your branching strategy.

In general, when you start working on a new branch, what you should do is:

```
git checkout master
git fetch exercism
git reset --hard exercism/master
git checkout -b <your-branch>
```

This will ensure that your newly created branch is based on the latest data of the master branch in the exercism remote.

**Comment by GalaDe** [comment]
Hi @ErikSchierboom, I know, I remember that you told me last time to create a separate PR for each change. Sorry, I messed up again (:  These were my steps:

1. git pull exersism master --> always check for updates. 
2. git add .    --> add your new commints to current commit
3. git status 
4. git commit -m "message" 
5. git push origin master --> push your repo from local machine to GitHub

I updated my local repo, but forgot to update my remote repo....

**Comment by ErikSchierboom** [comment]
@GalaDe Well, PR's are usually linked to a separate branch. It now looks like you commit to the master branch, which I would suggest you don't. If you look at my suggested workflow, what that does is to only use the master to branch from.

**Comment by ErikSchierboom** [comment]
Thanks! :tada:

**Comment by GalaDe** [comment]
Thank you @ErikSchierboom for explanation!

**Pull request merged**